ui/package-lock.json

```diff
@@ -17885,9 +17885,9 @@
       "integrity": "sha512-fBASbA6LnOU9dOU2eW7aQ8xmYBSXUIWr+UmF9b1efZBazGNO+rcXT/icdKnYm2pTwcRylVUYwW7H1PHfLekVzA=="
     },
     "react-query": {
-      "version": "3.33.7",
-      "resolved": "https://registry.npmjs.org/react-query/-/react-query-3.33.7.tgz",
-      "integrity": "sha512-uL/1ZatxcRyvfhbNPeuOlLVbcdMQKzQ2t+SQLJMrMM0plfa/Al3buaPbGsbsmIwvyWeI4CVkkSBKzQIkU4p9MQ==",
+      "version": "3.34.0",
```

Comment:
I guess we miss the change in `ui/package.json` ;)

And since we bump the version of React Query in shell UI, we have to check if it's still compatible with other federated UIs.